### PR TITLE
Added benchmarks.

### PR DESCRIPTION
**Pull request by n1k0**
r=? @gijsk @leibovic 

screen shot 2015-04-01 at 21 58 48

### CODE REVIEW

**Comment by gijsk** [comment]
Can we get an average of a fixed set of those ops/s so it's easier to compare the output ?

**Comment by gijsk** [comment]
(the total time for all benchmarks is less useful because new commits tend to add a new testcase)

**Comment by n1k0** [comment]
> Can we get an average of a fixed set of those ops/s so it's easier to compare the output ?

I couldn't find an option for this, except exporting the result as CSV (brrr). We could create a separate suite with a fixed subset of these test pages, and mark it as a constant reference point, though.

**Comment by gijsk** [comment]
>> Can we get an average of a fixed set of those ops/s so it's easier to compare the output ?

> I couldn't find an option for this, except exporting the result as CSV (brrr). We could create a separate suite with a fixed subset of these test pages, and mark it as a constant reference point, though.

That sounds like a good idea, yeah.

**Comment by n1k0** [comment]
screen shot 2015-04-01 at 22 27 37

**Comment by n1k0** [comment]
@gijsk now we have these two distinct commands, `npm run perf` and `npm run perf-reference`, but we need to select the [right subset of reference test pages](https://github.com/mozilla/readability/blob/33eb24f4070370d9778b53ecb337580d435586f6/benchmarks.js#L7-L12). Could you please take over this, then land the path when you're happy? It's a little late here :)

File: index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+var path = require("path");
+var fs = require("fs");
+var prettyPrint = require("html").prettyPrint;
+var chai = require("chai");
+chai.config.includeStack = true;
+var expect = chai.expect;
```
Comment (n1k0) [inline index.js]: Ah just spotted all this should be removed from this file, as we're not in a test context.

Comment (gijsk) [inline index.js]: Done.

**Pull request merged**